app/api/events.py

```diff
@@ -47,10 +47,10 @@ class Meta:
     location_name = fields.Str()
     searchable_location_name = fields.Str()
     description = fields.Str()
-    thumbnail_image_url = fields.Str(attribute='thumbnail')
-    large_image_url = fields.Str(attribute='large')
-    original_image_url = fields.Str(attribute='background_url')
-    icon_image_url = fields.Str(attribute='icon')
+    original_image_url = fields.Str()
```

Comment:
Actually there are still some issues in open event database where the proper urls are not stored. Changing it to Url will show error. This was suggested by Shubham to keep them Str() now in #3708